metaswitch/common/logging_config.py

```diff
@@ -50,26 +51,41 @@ def doRollover(self):
         self.stream = os.fdopen(os.open(self.baseFilename, os.O_WRONLY | os.O_CREAT, 0644), self.mode)
         self.next_file_change = (int(currentTime / 3600) * 3600) + 3600
 
+def configure_logging(log_level,
+                      log_dir,
+                      log_prefix,
+                      show_thread=False,
+                      **kwargs):
```

Comment:
You've changed the position of the `task_id` parameter here - was that deliberate?

If you want to keep this, I think you'll have to manually check everywhere we use the configure_logging function and make sure that `task_id` isn't used positionally, which it sounds like you've not done